tsconfig.json

```diff
@@ -1,22 +1,22 @@
 {
-    "compilerOptions": {
-      "target": "es6",
-      "module": "commonjs",
-      "outDir": "dist",
-      "rootDir": "./src/",
-      //Do not allow JavaScript files in the source
-      "allowJs": false,
-  
-      //Remove developer comments from the distributable code
-      "removeComments": true,
-  
-      //The following force the compiler to throw an error when their respective code smells are present
-      "noUnusedLocals": true,
-      "noImplicitReturns": true,
-      "noFallthroughCasesInSwitch": true
-    },
-    "exclude": [
-      "node_modules",
-      "client"
-    ]
-  }
\ No newline at end of file
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
What version of node do we need to make sure we have running for all ES 2017 syntax to be available?

Reply:
I noted this question only inside of this thread, but the question applies to each of the corresponding PRs. Just want to check and make sure the developer machines and our dock or containers are configured correctly. @nvisal1 @gshaw1997

Reply:
These sources state that ES 2017 syntax requires a minimum Node version of 6.5. 
https://node.green/
https://stackoverflow.com/questions/44027764/how-can-i-use-es7-in-nodejs

I believe all of our containers are running a minimum version of Node 8